### PR TITLE
Fix build error

### PR DESCRIPTION
**Pull request by ningyougang**
## Description

Fix build error, and the error is leaded by this pr: Upgrade to Akka 2.6.12: https://github.com/apache/openwhisk/pull/5065
When i tested, forgot to rebase that pr.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Scheduler
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [x] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by ddragosd** [comment]
@ningyougang thanks !

**Comment by ningyougang** [comment]
> @ningyougang thanks !

You are welcome.

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala [hunk 1]
```diff
@@ -715,7 +714,7 @@ class FunctionPullingContainerProxy(
       logging.info(
         this,
         s"This is the remaining container for ${data.action}. The container will stop after $warmedContainerKeepingTimeout.")
-      setTimer(KeepingTimeoutName, Remove, warmedContainerKeepingTimeout)
+      startSingleTimer(KeepingTimeoutName, Remove, warmedContainerKeepingTimeout)
```
Comment (ddragosd) [inline core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:717]: `startTimerAtFixedRate` instead ?

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala [hunk 2]
```diff
@@ -777,7 +776,7 @@ class FunctionPullingContainerProxy(
         }
       }
       unstashAll()
-    case _ -> Paused   => setTimer(IdleTimeoutName, StateTimeout, idleTimeout)
+    case _ -> Paused   => startSingleTimer(IdleTimeoutName, StateTimeout, idleTimeout)
```
Comment (ddragosd) [inline core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:779]: `startTimerAtFixedRate` instead ?

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala [hunk 3]
```diff
@@ -613,7 +612,7 @@ class FunctionPullingContainerProxy(
         logging.info(
           this,
           s"Remain running activations ${runningActivations.keySet().toString()} when received ClientClosed")
-        setTimer(RunningActivationTimeoutName, ClientClosed, runningActivationTimeout)
+        startSingleTimer(RunningActivationTimeoutName, ClientClosed, runningActivationTimeout)
```
Comment (ddragosd) [inline core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:615]: wondering if `startTimerAtFixedRate`​ should be used instead

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala (hunk 1 above)
Comment (ningyougang) [reply core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:717]: I checked the source code
```
  def startTimerAtFixedRate(name: String, msg: Any, interval: FiniteDuration): Unit =
    startTimer(name, msg, interval, FixedRateMode)

  ...
  /** INTERNAL API */
  @InternalApi
  private[akka] case object FixedRateMode extends TimerMode {
    override def repeat: Boolean = true
  }
```
the method: `startTimerAtFixedRate` will invoke startTimer with `repeat: true`,
but for here, it is a one time timer,
So startSingleTimer is ok here.

**Review by rabbah — COMMENTED** [review]
lgtm

**Review by rabbah — APPROVED** [review]

File: tests/src/test/scala/org/apache/openwhisk/core/containerpool/v2/test/FunctionPullingContainerProxyTests.scala [hunk 4]
```diff
@@ -1443,8 +1441,8 @@ class FunctionPullingContainerProxyTests
           Some(testContainerId)),
         ""))
     inAnyOrder {
-      probe.expectMsg(Transition(machine, Paused, Running))
       probe.expectMsgType[Resumed]
+      probe.expectMsg(Transition(machine, Paused, Running))
```
Comment (ningyougang) [inline tests/src/test/scala/org/apache/openwhisk/core/containerpool/v2/test/FunctionPullingContainerProxyTests.scala:1445]: Make the test case more stable.

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala (hunk 3 above)
Comment (ningyougang) [reply core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:615]: https://github.com/apache/openwhisk/pull/5123/files#r646383365

File: core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala (hunk 2 above)
Comment (ningyougang) [reply core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/v2/FunctionPullingContainerProxy.scala:779]: https://github.com/apache/openwhisk/pull/5123/files#r646383365

**Review by jiangpengcheng — APPROVED** [review]

**Pull request merged**